packages/venia-ui/lib/components/Checkout/index.js

```diff
@@ -1 +1,6 @@
+/**
+ * @deprecated and will be deleted in next release
```

Comment:
Looks like this never got deprecated, so we'll have to remove in next available release. Would have much preferred to just delete this component in this scope.

Reply:
I think we need to figure out a better deprecation story. For now we should just mark components as deprecated but I would avoid comments indicating deletion time frame going forward.

We don't want to alienate anyone who has done a lot of customization on their legacy checkout overrides.